devchat.go

```diff
@@ -252,6 +258,18 @@ func (u *user) close(msg string) {
 		u.session.Close()
 	})
 }
+func (u *user) system(message string) {
```

Comment:
This shouldn't be a method of `user`, please make it a function that takes in a user pointer.

Reply:
why? This is sending a system message to the user?

Reply:
As I said, we'd like the help messages to be broadcast

Reply:
Later we can add a command that excludes users from seeing command output

Reply:
I don't understand what exactly you mean when you resolve the convo without a reply here? Does it mean you'll do it? Not do it? Agree? Disagree?

Reply:
No i really disagree. it makes it hard to have actual conversations without losing track of the message you are reading because someone ran ./help.

Reply:
alright, let's keep command output private

Reply:
Alright :+1: